src/csvcubed/utils/tableschema.py

```diff
@@ -144,7 +141,7 @@ def load_json_ld_to_rdflib_graph(self) -> rdflib.ConjunctiveGraph:
             raise FailedToLoadRDFGraphException(self.csvw_metadata_file_path) from ex
 
 
-def add_triples_for_file_dependencies(
+def _add_triples_for_file_dependencies(
```

Comment:
Can you undo this rename? The function is exposed publicly for use by csvcubed-pmd.